labconnect/main/profile_routes.py

```diff
@@ -0,0 +1,120 @@
+from flask import Response, jsonify, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from sqlalchemy import select, delete
```

Comment:
you shouldn't use the select/delete functions from sqlalchemy directly. You should always use `db.select()` and `db.delete()` as flask handles things a little differently from my understanding